spotifycli/spotifycli.py

```diff
@@ -141,16 +167,22 @@ def show_status_position():
 
     # Both values are in microseconds
     position = datetime.timedelta(milliseconds=position_raw / 1000)
-    length = datetime.timedelta(milliseconds=metadata['mpris:length'][1] / 1000)
+    length = datetime.timedelta(
+        milliseconds=metadata['mpris:length'][1] / 1000
+    )
```

Comment:
Line wrapped for format, happy to revert this if you are OK with longer lines.

Reply:
This one is ok.